File: PRAK505-2510817220020-LailyAzizahIslami.py
def biodata(tahunlahir, A, B):
    tahunsekarang  = 2020
    umur = tahunsekarang - tahunlahir
    angkatan = tahunlahir + 18

    print("\nPerkenalkan Nama Saya       :" , A)
    print("Umur Saya                   :", umur)
    print("Saya Adalah Angkatan        :", angkatan)
    print("Asal Saya dari              :", B)

tahunlahir = int(input())
A = input()
B = input()

biodata(tahunlahir, A, B)

File: PRAK505-2510817220020-LailyAzizahIslami.c
#include <stdio.h> 
 
void Biodata(int tahunLahir, char A[], char B[]){ 
    int tahun_sekarang = 2020; 
    int umur = tahun_sekarang - tahunLahir;
    int angkatan = tahunLahir + umur;

    printf("\nPerkenalkan Nama Saya       : %s\n", A);
    printf("Umur Saya                   : %d\n", umur);
    printf("Saya Adalah Angkatan        : %d\n", angkatan);
    printf("Asal Saya dari              : %s\n", B);
}
 
int main() { 
  int tahunLahir; 
    char A[20], B[15]; 
    scanf(" %d",&tahunLahir); 
    scanf(" %[^\n]%*c",&A); 
    scanf(" %[^\n]%*c",&B); 
  Biodata(tahunLahir, A, B); 
  return 0; 
}